_posts/2018-02-06-unreliable-network.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: Quick Tips for App Developers on Surviving with Unreliable Network
+author: jarzka
+excerpt: Methods that you, as an application developer, can do to make your app feel more stable even if the underlying network is unreliable.
```

Comment:
I'd suggest "Things" instead of "Methods". Not sure you can really "do a method"